package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "hydraclient.js",
+    "version": "0.0.1",
+    "scripts": {
+        "test": "node node_modules/karma/bin/karma start",
+        "test-once": "node node_modules/karma/bin/karma start --single-run"
```

Comment:
Since `start` and `test` are standard script commands, I'd say just use them. It's also possible to chain script commands together, so I'd do the following:

```json
"scripts": {
    "start": "node node_modules/karma/bin/karma start",
    "test": "npm start -- --single-run"
}
```